iconservice/utils/bloom.py

```diff
@@ -19,12 +19,12 @@ def get_chunks_for_bloom(value_hash):
 
 
 def chunk_to_bloom_bits(chunk):
-    high, low = bytearray(chunk.encode())
+    high, low = chunk
```

Comment:
Recommend keeping original code.
```suggestion
    high, low = bytearray(chunk)
```

Reply:
Remove changes comment about chunk_to_bloom_bits() at the beginning of file

Reply:
Removed changes comment about chunk_to_bloom_bits() at the beginning of the file.

Reply:
Reverted to original bloom-filter code